src/views/DeprecatedManageList.jsx

```diff
@@ -0,0 +1,125 @@
+import { useState } from 'react';
+import { Navigate } from 'react-router-dom';
+import { addItem, shareList } from '../api';
+
+export function ManageListDNU({ listPath, currentUserId }) {
```

Comment:
1️⃣ This function should also be renamed to `DeprecatedManageList`, I assume!

2️⃣ For a bit of a small developer experience improvement, you could add a [JSDoc](https://jsdoc.app/about-getting-started) comment to `DeprecatedManageList` to mark it as deprecated.  The comment sits on the lines immediately above the name of the component, and it looks like this:

```js
/**
 * @deprecated
 */
```

This is just for your benefit and the benefit of anyone else reading the code: if a person's editor understands it, it'll decorate the component with a strike-through, showing them at a glance that the component is old and not to be used.